corehq/apps/app_manager/migrations/0030_credentialapplication.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.20 on 2025-08-01 11:06
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app_manager', '0029_delete_case_list_custom_variable_xml'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CredentialApplication',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('domain', models.CharField(max_length=255)),
+                ('app_id', models.CharField(max_length=255)),
+                ('activity_level', models.CharField(choices=[('1MON_ACTIVE', '1 Month'), ('2MON_ACTIVE', '2 Months'), ('3MON_ACTIVE', '3 Months'), ('6MON_ACTIVE', '6 Months'), ('9MON_ACTIVE', '9 Months'), ('12MON_ACTIVE', '12 Months')], default='3MON_ACTIVE', max_length=32)),
```

Comment:
Just FYI - if you expect this to change in future, it would need a migration to alter the choices.